This will open all files in binary mode.
files may contain binary data.
This fixes e.g. autoconf, of whose frozen file, contains all characters from 0x00 to 0xff.  

--- src/path.c
+++ src/path.c
@@ -111,7 +111,7 @@
 static FILE *
 m4_fopen (const char *file, bool binary)
 {
-  FILE *fp = fopen (file, binary ? "rbe" : "re");
+  FILE *fp = fopen (file, binary ? "rbe" : "rbe");
   if (fp)
     {
       struct stat st;
